lib/modules/platform/bitbucket/index.ts

```diff
@@ -314,7 +314,7 @@ export async function findPr({
       return null;
     }
 
-    return utils.prInfo(prs[0]);
+    return utils.prInfo(prs.sort((a, b) => b.id - a.id)[0]);
```

Comment:
This `if` clause fetches PRs where `state=open`. I think by definition there can be only one open PR per branch so this won't do anything

Reply:
Change line 321 to `.findLast()`